node-graph/gstd/src/memo.rs

```diff
@@ -72,8 +82,16 @@ impl<'i, T: 'i + Hash> Node<'i, Option<T>> for LetNode<T> {
 			None => &self.cache.iter().last().expect("Let node was not initialized").1,
 		}
 	}
+
+	fn reset(mut self: Pin<&mut Self>) {
+		if let Some(last) = std::mem::take(&mut self.cache).into_iter().last() {
+			self.cache = boxcar::vec![last];
+		}
+	}
 }
 
+impl<T> std::marker::Unpin for LetNode<T> {}
+
 impl<T> LetNode<T> {
 	pub fn new() -> LetNode<T> {
 		LetNode { cache: boxcar::Vec::new() }
```

Comment:
This is actually the most relevant part, we can basically replace the box car with a single unsafe cell because this should during the runtime of the graph only ever have one value which can be thrown away at the end

Reply:
Nevermind didn't see this was already implemented for the cache node